code/_onclick/hud/new_player.dm

```diff
@@ -11,10 +11,7 @@
 /datum/hud/new_player/New(mob/owner)
 	. = ..()
 
-	if (!owner || !owner.client)
-		return
-
-	if (owner.client.interviewee)
+	if (!owner?.client || owner.client.interviewee)
 		return
```

Comment:
ftr there is no risk of going null here past the guard check. clients do not go null randomly. this is still fine but like

Reply:
it's deterministic behavior it's just not the most clear/easy to work with.

Reply:
@LemonInTheDark is there a writeup/documentation anywhere on when/how they go null?